rollup.config.js

```diff
@@ -8,7 +8,14 @@ import svg from 'rollup-plugin-svg';
 
 export default {
   // Output all components to a single exported file.
-  input: 'src/components/**/*[!(stories|lit)].ts',
+  input: {
+    include: ['src/components/**/*.ts'],
+    exclude: [
+      'src/components/**/*.stories.ts',
+      'src/components/**/*.css.lit.ts',
+      'src/components/**/*.test.ts',
```

Comment:
This worked for me. We will want to though in conjunction with other work most likely by default, exclude the code examples as we don't want to actually send out `extended-element`, etc. out to the consumer. They would still be visible in Storybook, but not part of the production build that Rollup powers. 

We can discuss that on #75 and what our examples on that page look like.